pkgs/by-name/pi/pixel-code/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
+  name = "pixel-code";
+  version = "2.1";
+
+  src = fetchzip {
+    url = "https://github.com/qwerasd205/PixelCode/releases/download/v${version}/otf.zip";
+    hash = "sha256-qu55qXcDL6YIyiFavysI9O2foccvu2Hyw7/JyIMXYv4=";
+    stripRoot=false;
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D -m444 -t $out/share/fonts/opentype $src/otf/*.otf
```

Comment:
```suggestion
    install -D -m444 -t $out/share/fonts/opentype/PixelCode $src/otf/*.otf
```

There should be one more path for them, I think, at least on my system there is:

```
/nix/store/znivrin5aj1034iwqllkim69qnc7his4-home-manager-path/share/fonts/opentype:
NerdFonts  source-han-sans

=> ls /nix/store/znivrin5aj1034iwqllkim69qnc7his4-home-manager-path/share/fonts/opentype/source-han-sans
SourceHanSans.ttc

=> ls /nix/store/znivrin5aj1034iwqllkim69qnc7his4-home-manager-path/share/fonts/opentype/NerdFonts
GeistMonoNerdFont-Black.otf      GeistMonoNerdFontMono-Medium.otf      GeistMonoNerdFontPropo-Bold.otf        GeistMonoNerdFontPropo-UltraLight.otf
GeistMonoNerdFont-Bold.otf       GeistMonoNerdFontMono-Regular.otf     GeistMonoNerdFontPropo-Light.otf       GeistMonoNerdFont-Regular.otf
GeistMonoNerdFont-Light.otf      GeistMonoNerdFontMono-SemiBold.otf    GeistMonoNerdFontPropo-Medium.otf      GeistMonoNerdFont-SemiBold.otf
GeistMonoNerdFont-Medium.otf     GeistMonoNerdFontMono-Thin.otf        GeistMonoNerdFontPropo-Regular.otf     GeistMonoNerdFont-Thin.otf
GeistMonoNerdFontMono-Black.otf  GeistMonoNerdFontMono-UltraBlack.otf  GeistMonoNerdFontPropo-SemiBold.otf    GeistMonoNerdFont-UltraBlack.otf
GeistMonoNerdFontMono-Bold.otf   GeistMonoNerdFontMono-UltraLight.otf  GeistMonoNerdFontPropo-Thin.otf        GeistMonoNerdFont-UltraLight.otf
GeistMonoNerdFontMono-Light.otf  GeistMonoNerdFontPropo-Black.otf      GeistMonoNerdFontPropo-UltraBlack.otf
```

Reply:
I'm guessing that does not make or break things. I was able to test this font on my system using the current install location and I see a lot of other font packages doing the same thing (for example https://github.com/nixos/nixpkgs/blob/master/pkgs/data/fonts/raleway/default.nix#L16..L16).